### PR TITLE
Move code

### PR DESCRIPTION
**Pull request by snopoke**
@benrudolph  @czue @dannyroberts  @NoahCarnahan 
:blowfish: 

cc @biyeun  @calellowitz

### CODE REVIEW

File: corehq/ex-submodules/couchforms/__init__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 from .exceptions import XMLSyntaxError, CouchFormException, UnexpectedDeletedXForm
 from .getters import *
-from .util import SubmissionPost, fetch_and_wrap_form
+from .util import fetch_and_wrap_form
```
Comment (dimagimon) [inline corehq/ex-submodules/couchforms/__init__.py]: F401 'fetch_and_wrap_form' imported but unused

File: corehq/form_processor/parsers/form.py [hunk 2]
```diff
@@ -0,0 +1,185 @@
+import hashlib
+
+from corehq.form_processor.interfaces.dbaccessors import FormAccessors
+from corehq.form_processor.interfaces.processor import FormProcessorInterface
+from corehq.form_processor.models import Attachment
+from corehq.form_processor.utils import convert_xform_to_json, adjust_datetimes
+from couchforms import XMLSyntaxError
+from couchforms.exceptions import SubmissionError, DuplicateError
```
Comment (dimagimon) [inline corehq/form_processor/parsers/form.py]: F401 'SubmissionError' imported but unused

File: corehq/form_processor/submission_post.py [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+
+import couchforms
+from casexml.apps.case.exceptions import PhoneDateValueError, IllegalCaseId, UsesReferrals
+from corehq.apps.commtrack.exceptions import MissingProductId
+from corehq.apps.tzmigration import timezone_migration_in_progress
+from corehq.form_processor.interfaces.processor import FormProcessorInterface
+from corehq.form_processor.parsers.form import process_xform_xml
+from corehq.form_processor.utils.metadata import scrub_meta
+from couchforms.const import BadRequest, DEVICE_LOG_XMLNS
+from couchforms.exceptions import SubmissionError
```
Comment (dimagimon) [inline corehq/form_processor/submission_post.py]: F401 'SubmissionError' imported but unused

**Comment by dannyroberts** [comment]
Looks related?
```
======================================================================
ERROR: testSubmitBadXML (corehq.apps.receiverwrapper.tests.test_submit_errors.SubmissionErrorTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/util/test_utils.py", line 169, in inner
    return helper(*args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/util/test_utils.py", line 142, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/util/test_utils.py", line 157, in call_with_settings
    fn(*args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/util/test_utils.py", line 138, in fn_with_pre_and_post
    self.fn(*args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/receiverwrapper/tests/test_submit_errors.py", line 105, in testSubmitBadXML
    limit=1)
ValueError: need more than 0 values to unpack
```

**Comment by dannyroberts** [comment]
nice changes, a few questions

**Comment by biyeun** [comment]
buildfail looks real?

**Comment by dannyroberts** [comment]
:+1: once tests pass

**Comment by snopoke** [comment]
✅

**Comment by dannyroberts** [comment]
nice re:factory:ing

**Pull request merged**